packages/bsky/service/api.js

```diff
@@ -65,9 +66,19 @@ const main = async () => {
   })
   const viewMaintainer = new ViewMaintainer(migrateDb)
   const viewMaintainerRunning = viewMaintainer.run()
+
+  const periodicModerationActionReversal = new PeriodicModerationActionReversal(
+    pds.ctx,
```

Comment:
I think we want this to be:
```suggestion
    bsky.ctx,
```